ironfish/src/rpc/routes/wallet/exportAccount.ts

```diff
@@ -13,6 +13,7 @@ export type ExportAccountResponse = {
     incomingViewKey: string
     outgoingViewKey: string
     publicAddress: string
```

Comment:
We probably want to move this version stuff to a separate PR. It would be nice to write some tests to verify the version handling is working as expected (for future proofing when we go to v2)

Reply:
Opened as #3222